dev/examples/controls/bar.js

```diff
@@ -415,18 +414,16 @@ export const renderBarWithPanning = (id) => {
             }
         ]
     };
-    const createGraph = (axis, values) => {
-        if (graph) {
-            graph.destroy();
-        }
-        graph = Carbon.api.graph(axis);
-        graph.loadContent(Carbon.api.bar(values));
-        return graph;
+    const createGraph = () => {
+        graph.reflow();
```

Comment:
Instead of not loading any additional data when clicking on next and previous can we make up data values?